### PR TITLE
added model, dao and service for ShoppingCart and Ticket

### PR DESCRIPTION
**Pull request by Oleh888**

### CODE REVIEW

File: src/main/java/com/dev/cinema/model/Ticket.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table (name = "ticket")
+public class Ticket {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToOne
```
Comment (boroda4436) [inline src/main/java/com/dev/cinema/model/Ticket.java:17]:
```suggestion
    @ManyToOne
```

File: src/main/java/com/dev/cinema/model/Ticket.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    private Movie movie;
+    @OneToOne
```
Comment (boroda4436) [inline src/main/java/com/dev/cinema/model/Ticket.java:19]:
```suggestion
    @ManyToOne
```

File: src/main/java/com/dev/cinema/model/Ticket.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    private CinemaHall cinemaHall;
+    private LocalDateTime showTime;
+    @OneToOne
```
Comment (boroda4436) [inline src/main/java/com/dev/cinema/model/Ticket.java:22]:
```suggestion
    @ManyToOne
```

File: src/main/java/com/dev/cinema/security/AuthenticationServiceImpl.java [hunk 4]
```diff
@@ -4,11 +4,14 @@
...
 import com.dev.cinema.lib.Service;
 import com.dev.cinema.model.User;
+import com.dev.cinema.service.ShoppingCartService;
 import com.dev.cinema.service.UserService;
 import com.dev.cinema.util.HashUtil;
 
 @Service
 public class AuthenticationServiceImpl implements AuthenticationService {
+    @Inject
+    ShoppingCartService shoppingCartService;
```
Comment (boroda4436) [inline src/main/java/com/dev/cinema/security/AuthenticationServiceImpl.java:14]: access modifier

Comment (Oleh888) [reply src/main/java/com/dev/cinema/security/AuthenticationServiceImpl.java:14]: done

**Pull request merged**